libraries/core-react/stories/testing/GlobalStyles.ts

```diff
@@ -0,0 +1,182 @@
+// Testing global variables when migrating
+
+import { createGlobalStyle } from 'styled-components'
+
+export const GlobalStyleDark = createGlobalStyle`
+:root {
```

Comment:
Can we interpolate this string with reading the tokens.css content directly? easier to update changes from figma-broker instead of copy paste content from one file to another?

Reply:
Not for the moment at least, but probably when we have added variables to the library. It's just for testing purposes anyway, so no need to overcomplicate things